readme.md

```diff
@@ -95,6 +101,9 @@ Use `docker network create --attachable "{YOUR_NETWORK_NAME}"` once to create
 the network. Don't change it afterwards. You can now test if the network was created with  
 `docker network ls`.
 
+Don't forget to add `127.0.0.1 docker.test` to your `/etc/hosts` (`sudo` necessary) file,
+which docker will use to redirect the requests on your local machine.
+
```

Comment:
Isn't this unnecessary now?

Reply:
I don't think so. If you do not wish to use the `DockerExec`, you machine still needs to know the domains of your dev network. So we better keep the annotation.